features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -108,6 +108,12 @@
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.dsmr/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-binding-enocean" description="EnOcean Binding" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
+        <feature>openhab-transport-serial</feature>
+        <feature>esh-config-discovery-usbserial</feature>
```

Comment:
This is already included in `openhab-transport-serial`, so you can remove this line.

Reply:
fixed